### PR TITLE
Docs/icon config examples

### PR DESCRIPTION
**Pull request by m0ksem**
Removed a lot of junk text that prevents user from reading I believe :) 
Added interactive example, so user can play with config.

closes https://github.com/epicmaxco/vuestic-ui/issues/2079

image
image
image

image

### CODE REVIEW

**Review by rustem-nasyrov — APPROVED** [review]

File: packages/docs/src/components/demo/CodeInput.vue [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+const valueComputed = computed({
+  get () { return props.modelValue },
+  set (v) { emit('update:modelValue', v) },
+})
+
+const textDiv = ref<HTMLDivElement>()
+const textDivWidth = ref(0)
+const inputPaddingX = 4
+const inputPaddingXPx = `${inputPaddingX}px`
```
Comment (RVitaly1978) [inline packages/docs/src/components/demo/CodeInput.vue:26]:
```suggestion
```
I think you've been practicing with `v-bind` in scss))

File: packages/docs/src/components/demo/CodeInput.vue [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  .code-input {
+    color: currentColor;
+    width: min-content;
+    position: relative;
+    display: inline-block;
+
+    &__input {
+      position: relative;
+      border: none;
+      padding: 0px v-bind(inputPaddingXPx);
```
Comment (RVitaly1978) [inline packages/docs/src/components/demo/CodeInput.vue:49]:
```suggestion
      padding: 0px 4px;
```

File: packages/docs/src/page-configs/services/icons-config/components/examples.ts [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    },
+  },
+  {
+    label: 'Material icons',
+    value: {
+      name: 'mdi-{icon}',
+      resolve: {
+        class: '\'material-icons\'',
+        content: 'icon',
+      },
```
Comment (RVitaly1978) [inline packages/docs/src/page-configs/services/icons-config/components/examples.ts:33]:
```suggestion
        content: 'icon',
        tag: "'span'",
      },
```

**Review by rustem-nasyrov — APPROVED** [review]

File: packages/docs/src/page-configs/services/icons-config/components/playground.vue [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+    .entries(resolve)
+    .filter(([key, value]) => Boolean(value))
+    .map(([key, value]) => `${' '.repeat(tab)}${key}: ${value},`)
+    .join('\n')
+}
+
+const renderConfig = () => {
+  return `const fonts = [{
+  name: '${configName.value}',
+  resolve: ({ ${args.value} }) => ({
```
Comment (RVitaly1978) [inline packages/docs/src/page-configs/services/icons-config/components/playground.vue:105]:
```suggestion
  resolve: (${args.value}) => ({
```

File: packages/docs/src/page-configs/services/icons-config/components/use-copy-to-clipboard.ts [hunk 5]
```diff
@@ -0,0 +1,23 @@
+import { useToast } from 'vuestic-ui/src/main'
+import { getWindow } from 'vuestic-ui/src/utils/ssr-utils'
+
+export const useCopyToClipboard = () => {
+  const { init } = useToast()
+
+  const copy = (text: string) => {
+    try {
+      getWindow()?.navigator.clipboard.writeText(text)
```
Comment (RVitaly1978) [inline packages/docs/src/page-configs/services/icons-config/components/use-copy-to-clipboard.ts:9]:
```suggestion
  const copy = async (text: string) => {
    try {
      await getWindow()?.navigator.clipboard.writeText(text)
```

**Review by RVitaly1978 — CHANGES_REQUESTED** [review]

File: packages/docs/src/components/demo/CodeInput.vue (hunk 1 above)
Comment (m0ksem) [reply packages/docs/src/components/demo/CodeInput.vue:26]: I thought I will need these values in calculation.

**Review by asvae — APPROVED** [review]
Awesome!

**Pull request merged**